### PR TITLE
Click the editor timeline to skip to that part in the video

### PR DESCRIPTION
**Pull request by albinekb**
kapture 2017-10-24 at 11 34 30

### CODE REVIEW

**Comment by albinekb** [comment]
Sorry for marking everyone as “reviewers” (forgot github sent out emails and stuff  :see_no_evil: )

File: app/src/renderer/js/editor.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ document.addEventListener('DOMContentLoaded', () => {
   const windowHeader = $('.window-header');
   const trimmerIn = $('#trimmer-in');
   const trimmerOut = $('#trimmer-out');
+  const trimline = $('.timeline-markers');
```
Comment (sindresorhus) [inline app/src/renderer/js/editor.js]: Nitpick, but `trimline` is not a single word, so should be `trimLine`.

File: app/src/renderer/js/editor.js [hunk 2]
```diff
@@ -69,6 +70,8 @@ document.addEventListener('DOMContentLoaded', () => {
 
   playBtn.onclick = play;
 
+  trimline.onclick = skip;
```
Comment (sindresorhus) [inline app/src/renderer/js/editor.js]: `addEventListener()`

File: app/src/renderer/js/editor.js [hunk 3]
```diff
@@ -81,6 +84,21 @@ document.addEventListener('DOMContentLoaded', () => {
     preview.play();
   }
 
+  function skip(event) {
+    const rect = event.target.getBoundingClientRect();
+    const x = event.pageX - rect.left; // X position within the trimmer
```
Comment (sindresorhus) [inline app/src/renderer/js/editor.js]: Just name the variable `xPositionInTrimmer` and you don't need the comment.

File: app/src/renderer/js/editor.js [hunk 4]
```diff
@@ -81,6 +84,21 @@ document.addEventListener('DOMContentLoaded', () => {
...
+    const skipTime = preview.duration * (x / rect.width); // Calculated time in seconds where the click happened
+
+    if (skipTime < getTrimmerValue(trimmerIn)) {
+      return;
+    } // Clicked before the first trimmer
```
Comment (sindresorhus) [inline app/src/renderer/js/editor.js]: Put the comment above the block and add an empty line between the if statements.

**Comment by sindresorhus** [comment]
The code looks good except for some nitpick.

**Comment by albinekb** [comment]
Closing this in favour of #245

**Pull request closed without merging**